docs/client_reference.rst

```diff
@@ -524,29 +527,41 @@ The client session supports the context manager protocol for self closing.
       :param aiohttp.BasicAuth proxy_auth: an object that represents proxy HTTP
                                            Basic Authorization (optional)
 
-      :param int compress: Enable Per-Message Compress Extension support.
-                           0 for disable, 9 to 15 for window bit support.
-                           Default value is 0.
+      :param bool verify_ssl: Perform SSL certificate validation for
+         *HTTPS* requests (enabled by default). May be disabled to
+         skip validation for sites with invalid certificates.
 
-      .. versionadded:: 0.16
+         .. versionadded:: 2.3
 
-         Add :meth:`ws_connect`.
+      :param bytes fingerprint: Pass the SHA256 digest of the expected
+           certificate in DER format to verify that the certificate the
+           server presents matches. Useful for `certificate pinning
+           <https://en.wikipedia.org/wiki/Transport_Layer_Security#Certificate_pinning>`_.
 
-      .. versionadded:: 0.18
+           Note: use of MD5 or SHA1 digests is insecure and deprecated.
 
-         Add *auth* parameter.
```

Comment:
Did you intentionally removed part of the documentation?

Reply:
Yes.
I've dropped `versioadded` records for versions < 2.0
They don't carry any useful information now.

Reply:
Yup it was quite noisy after all.